app/components/ChangeUser.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { ScrollView, StyleSheet, TextStyle, View } from "react-native";
+import { LinkedAddress, UserDetail } from "../models/User";
+import { changeUser } from "../services/ApiService";
+import NotificationService from "../services/NotificationService";
+import SessionService from "../services/SessionService";
+import { Text, TouchableRipple } from "react-native-paper";
+import Colors from "../config/Colors";
+
+interface Props {
+  user?: UserDetail;
+  onSubmit: () => void;
```

Comment:
Why onSubmit? 😉

Reply:
no special reasoning, i can rename it to something else :D